pkgs/applications/version-management/git-and-tools/default.nix

```diff
@@ -75,11 +75,15 @@ let
   git-bug = callPackage ./git-bug { };
 
   # support for bugzilla
-  git-bz = callPackage ./git-bz { };
+  git-bz = callPackage ./git-bz {
+    pythonPackages = python2Packages;
+  };
 
   git-codeowners = callPackage ./git-codeowners { };
 
-  git-cola = callPackage ./git-cola { };
+  git-cola = callPackage ./git-cola {
+    pythonPackages = python2Packages;
```

Comment:
Rather than adding an override in `all-packages.nix` it's better to change the name of the function argument to `python2Packages` so no override is necessary at all.